composer.json

```diff
@@ -24,7 +24,9 @@
     },
     "require-dev": {
         "league/oauth2-facebook": "^1.1",
-        "symfony/security-guard": "^2.8|^3.0"
+        "symfony/security-guard": "^2.8|^3.0",
+        "phpunit/phpunit": "^4.8",
```

Comment:
I'm fine with using just one major, as this - because being a `dev` dependency, this won't cause any problems with end-user projects, correct (i.e. an end-user could still install `phpunit/phpunit` version 5 in their project, without this conflicting)?

And, is there a precedence for including `php-code-coverage` as a dependency in other OS projects? I'm just not familiar with this - and mostly, I want to follow what's most expected/standard.

Thanks!